### PR TITLE
9700 duplicate work items to test

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: shared/src/persistence/dynamo/locks/acquireLock.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import { getTableName } from '../../dynamodbClientService';
+
+/**
+ * will wrap a function with logic to acquire a lock and delete a lock after finishing.
+ */
+export function withLocking(
```
Comment (ujahio) [inline shared/src/persistence/dynamo/locks/acquireLock.ts:6]: @codyseibert 
❓ : Do these util functions need jsdocs write up?

**Review by ujahio — APPROVED** [review]

File: web-client/src/presenter/sequences/completeDocketEntryQCAndSendMessageSequence.ts [hunk 2]
```diff
@@ -28,17 +28,22 @@ export const completeDocketEntryQCAndSendMessageSequence = [
...
-      setSaveAlertsForNavigationAction,
-      setCaseAction,
-      setAlertSuccessAction,
-      getMessagesForCaseAction,
-      setPdfPreviewUrlAction,
-      setPaperServicePartiesAction,
-      navigateToDocumentQCAction,
-      clearFormAction,
+      {
+        error: [],
```
Comment (rosiehsmith) [inline web-client/src/presenter/sequences/completeDocketEntryQCAndSendMessageSequence.ts:32]: should this show a modal or anything on error?

File: shared/src/persistence/dynamo/locks/acquireLock.ts
Comment (rosiehsmith) [inline shared/src/persistence/dynamo/locks/acquireLock.ts:1]: 🙌 nice

File: shared/src/business/useCases/editDocketEntry/completeDocketEntryQCInteractor.ts [hunk 3]
```diff
@@ -376,3 +377,10 @@ export const completeDocketEntryQCInteractor = async (
     paperServicePdfUrl,
   };
 };
+
+export const completeDocketEntryQCInteractor = withLocking(
```
Comment (rosiehsmith) [inline shared/src/business/useCases/editDocketEntry/completeDocketEntryQCInteractor.ts:381]: test?

**Review by rosiehsmith — APPROVED** [review]

File: web-client/src/presenter/sequences/completeDocketEntryQCAndSendMessageSequence.ts (hunk 2 above)
Comment (codyseibert) [reply web-client/src/presenter/sequences/completeDocketEntryQCAndSendMessageSequence.ts:32]: I'll add something

**Pull request merged**